packages/mobile/src/harmony-native/components/input/TextInput/types.ts

```diff
@@ -31,6 +31,7 @@ type InternalProps = {
 }
 
 export type TextInputProps = RNTextInputProps & {
+  id?: string
```

Comment:
is this related?

Reply:
i noticed that npm run typecheck complains so no, i ultimately just fixed in another pr, so will merge here and should go away